packages/richtext-lexical/src/lexical/theme/EditorTheme.scss

```diff
@@ -221,15 +221,17 @@
 
   &__listItem {
     font-size: base(0.8);
-    margin: 0 0 0.4em base(0.8);
+    margin: 0 0 0.4em 40px;
   }
 
   &__listItemChecked,
   &__listItemUnchecked {
     position: relative;
-    margin-inline: 0;
-    padding-left: base(1.2);
-    padding-right: base(1.2);
+    // Instead of having margin-left: 40px like other list-items or indented paragraphs, 
+    // we use padding-left: 25px + margin-left: 15px = 40px, so that the click position 
+    // calculation in `CheckListPlugin` matches the checkbox
+    margin-left: 15px;
+    padding-left: 25px;
```

Comment:
We should consider using `margin-inline-start` and `padding-inline-start` to acommodate for RTL languages.

Reply:
Good observation! Instead of using `margin-inline-start` and `padding-inline-start` I used the selector `[dir='rtl']`, so as not to have unnecessary margins or paddings on the opposite side.

![image](https://github.com/user-attachments/assets/5b85812c-e6e9-4337-9bb5-dd5da6dc02a1)

What do you think now?